cmake/external/StanMath.cmake

```diff
@@ -0,0 +1,26 @@
+set(STAN_STABLE_RELEASE 4b1a10bc877d941bbe0a12c198526807be27167a)
+
+include(ExternalProject)
+ExternalProject_Add(
+	StanMath
+	PREFIX ${CMAKE_BINARY_DIR}/StanMath
+	DOWNLOAD_DIR ${THIRD_PARTY_DIR}/StanMath
+	GIT_REPOSITORY https://github.com/stan-dev/math
+	GIT_TAG ${STAN_STABLE_RELEASE}
+	CONFIGURE_COMMAND ""
+	BUILD_COMMAND ""
+	INSTALL_COMMAND ""
+	LOG_DOWNLOAD ON
+	)
+
+
+add_dependencies(libshogun StanMath)
+
+set(STAN_INCLUDE_DIR_STAN_MATH ${CMAKE_BINARY_DIR}/StanMath/src/StanMath)
+set(STAN_INCLUDE_DIR_BOOST ${CMAKE_BINARY_DIR}/StanMath/src/StanMath/lib/boost_1.64.0)
+set(STAN_INCLUDE_DIR_CVODES ${CMAKE_BINARY_DIR}/StanMath/src/StanMath/lib/cvodes_2.9.0/include)
+
+
+INSTALL( DIRECTORY ${STAN_INCLUDE_DIR_STAN_MATH} DESTINATION include/shogun/lib/external/Stan )
```

Comment:
instead of this you should actually copy these files into `${THIRD_PARTY_INCLUDE_DIR}/Stan`

Reply:
remove these installs.... if you do a cmake copy thing then you will not have problems with the logs in travis.

Reply:
check how `COMMAND ${CMAKE_COMMAND} -E copy_if_different` is being used in shogun

Reply:
and the best place to do this copy would be `ExternalProject_Add`'s `INSTALL_COMMAND`